src/auto/accel_group.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (b7f5189) from gir-files (71d73f0)
+// This file was generated by gir (b7f5189) from gir-files (71d73f0+)
```

Comment:
Hash saved to `src/gir_version.rs` but sometimes build silently fails to do that (or think that nothing changed or just don't run `build.rs`)